_posts/2021-01-11-GPS-serverless.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: 7 Steps to Create a Serverless Tracking Solution
+date: 2021-01-11
+tags: cloudformation serverless appsync dynamodb
+author: Shermayne Lee
+image:
+---
+
+While being a dog lover, I love spending most of my free time outdoors with my dog. However, I always have difficulty calling him back when he runs outside, and I hate when he is out of my sight. Therefore to solve a few challenges I faced when I am with him, I came out with a serverless GPS monitoring solution to track his movement.
+
+## Solution ?
+
+<center><img src="/img/blog/serverless-GPS/map.png" /></center><br/>
+
+<center><img src="/img/blog/serverless-GPS/archi.jpeg" /></center><br/>
+
+## First Step: Getting started with the physical devices
+
+In my opinion, this could be the most challenging part because there are so many devices available in the market, and I have no idea which is the right one for my solution.
+
+My rule of thumb is always getting something more comfortable to start. For instance, get a physical device that is fully assembled instead of getting every separate part and try to figure out how to build it.
+
+Of course, if you have a vast experience and interest in devices and parts, feel free to start from scratch in building a tracking device.
+
+My focus in this post is on sharing my experience building a serverless tracking solution on the cloud.
+
+In my solution, I have a Dragino-LGT-92 tracking device. It is a handheld LORAWAN device with real-time GPS tracking and motion sensing capability. It is an open-source GPS tracker base on ultra-low-power Lora module.
+
+## Second Step: Register the LORA tracking device
+
+After I bought a LoRaWAN tracking device, I need a platform to manage my device and connect the device to the internet.
+
+Before I start, let's understand a few terms I will frequently use in this post.
```

Comment:
There is a lot of "I did this" "I did that" in this post. You can tell a story using what you did but also make it more applicable to the reader. What will THEY need to do if they follow this post?